### PR TITLE
updated yt links to bytesize talks to show on the website

### PR DESCRIPTION
**Pull request by FranBonath**
I realised the youtube links I added before did not show on the website. They will now (I hope). Also added the links for the newest nanoseq bytesize talk.

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1195"></a>

### CODE REVIEW

**Review by FriederikeHanssen — APPROVED** [review]

**Comment by FriederikeHanssen** [comment]
oh the linting fails on some dead links

File: markdown/events/2022/bytesize-cutandrun.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ end_time: '13:30 CEST'
 youtube_embed: https://www.youtube.com/watch?v=rj5i9deNPHA
 location_url:
   - https://doi.org/10.6084/m9.figshare.19923428.v1
+  - https://www.youtube.com/watch?v=kBoC6QBU-M0
```
Comment (ewels) [inline markdown/events/2022/bytesize-cutandrun.md:12]: This is for gitpod 😉

Comment (ewels) [reply markdown/events/2022/bytesize-cutandrun.md:12]:
```suggestion
  - https://www.youtube.com/watch?v=rj5i9deNPHA
```

**Review by ewels — APPROVED** [review]
Awesome, thanks 👍🏻

**Comment by ewels** [comment]
> oh the linting fails on some dead links

Yeah it's been failing for ages on all PRs. Needs solving.

**Pull request merged**